reference/gzip.md

```diff
@@ -16,7 +16,7 @@ For more details see [Envoy - Gzip](https://www.envoyproxy.io/docs/envoy/latest/
 - `compression_strategy`: A value used for selecting the zlib compression strategy which is directly related to the characteristics of the content. Most of the time “DEFAULT” will be the best choice, though there are situations in which changing this parameter might produce better results. For example, run-length encoding (RLE) is typically used when the content is known for having sequences in which the same data occurs many consecutive times. For more information about each strategy, please refer to the zlib manual.
 - `window_bits`: A value from 9 to 15 that represents the base two logarithmic of the compressor’s window size. Larger window results in better compression at the expense of memory usage. The default is 12 which will produce a 4096 bytes window. For more details about this parameter, please refer to zlib manual > deflateInit2.
 - `content_type`: A set of strings that specify which mime-types yield compression; e.g., application/json, text/html, etc. When this field is not defined, compression will be applied to the following mime-types: “application/javascript”, “application/json”, “application/xhtml+xml”, “image/svg+xml”, “text/css”, “text/html”, “text/plain”, “text/xml”.
-- `disable_on_etag_header`: A Boolean, if true, disables compression when the response contains an `etag` header. When it is false, the filter will preserve weak `etag`s and remove the ones that require strong validation.
+- `disable_on_etag_header`: A Boolean, if true, disables compression when the response contains an ETag header. When it is false, the filter will preserve weak ETags and remove the ones that require strong validation.
```

Comment:
FYI, I cheated. I wrote this as "ETag" (which is the proper way of writing it) when resolving merge conflicts for you.